src/stringIndicies.js

```diff
@@ -0,0 +1,11 @@
+function stringIndicies(input, index){
```

Comment:
I am having some issues running this and think there's some syntax issues. Did this work for you? 
I refactored it to this and it worked: `let stringIndicies = (input, index) => {
  let toArray = input.split(' ')
  if(index >= toArray.length){
    return ""
  } else {
    return toArray[index - 1]
  }
}`